ui/src/kapacitor/components/AlertTabs.tsx

```diff
@@ -443,7 +453,7 @@ class AlertTabs extends PureComponent<Props, State> {
   }
 
   private getConfigEnabled = (sections: Sections, section: string): boolean => {
-    if (section === AlertTypes.slack) {
+    if (section === AlertTypes.slack || section === AlertTypes.kafka) {
```

Comment:
what do you think about making a list, such as a const array or enum, of all possible node types that allow for multiple configs, and having a function check for presence there? i'm not sure how to say this in a more abstract way... but maybe like, making a functional guard instead of a logical guard?

Reply:
I think that is a good idea when we have more places to check, right now this is the only place that we check in a general way and everywhere else is looking for a specific alert type.